### PR TITLE
docstrings update in model_builder.py

### PR DESCRIPTION
**Pull request by michaelraczycki**
Updates the docstrings in model_builder.py , ensures that they follow the numpydoc conventions

### CODE REVIEW

File: pymc_experimental/model_builder.py [hunk 1]
```diff
@@ -131,9 +131,15 @@ def create_sample_input():
         >>>       'chains': 1,
         >>>       'target_accept': 0.95,
         >>>    }
-        >>>    return data, model_config, sampler_config
```
Comment (twiecki) [inline pymc_experimental/model_builder.py:134]: I think this should stay.

Comment (michaelraczycki) [reply pymc_experimental/model_builder.py:134]: oh, my bad. I was quite sure I added the Return section without deleting the example

Comment (michaelraczycki) [reply pymc_experimental/model_builder.py:134]: it's back again

**Comment by twiecki** [comment]
Conflict, but LGTM otherwise.

**Review by twiecki — APPROVED** [review]

**Pull request merged**